src/NServiceBus.Core/Routing/FileBasedDynamicRouting/FileRoutingTableFeature.cs

```diff
@@ -11,19 +12,31 @@ public FileRoutingTableFeature()
             {
                 s.SetDefault(CheckIntervalSettingsKey, TimeSpan.FromSeconds(30));
                 s.SetDefault(MaxLoadAttemptsSettingsKey, 10);
+                s.SetDefault(FilePathSettingsKey, "instance-mapping.xml");
             });
             DependsOn<RoutingFeature>();
         }
 
         protected internal override void Setup(FeatureConfigurationContext context)
         {
             var filePath = context.Settings.Get<string>(FilePathSettingsKey);
+            if (!Path.IsPathRooted(filePath))
+            {
+                filePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, filePath);
+            }
+
+            if (!File.Exists(filePath))
```

Comment:
Do we actually want to return here if the file doesn't exist? This means the file has to exist at startup, or it will never be checked for again. If we allow it to continue, then the file could be picked up as soon as it exists without needing to restart the endpoint.